stream-chat-android-ui-utils/src/main/kotlin/io/getstream/chat/android/uiutils/constant/AttachmentType.kt

```diff
@@ -14,14 +14,19 @@
  * limitations under the License.
  */
 
-package io.getstream.chat.android.compose.ui.util.extensions.internal
-
-import io.getstream.chat.android.client.ChatClient
-import io.getstream.chat.android.compose.state.userreactions.UserReactionItemState
+package io.getstream.chat.android.uiutils.constant
 
 /**
- * @return if the current reaction is owned by current user or not.
+ * Represents types of attachments.
  */
-internal fun UserReactionItemState.isMine(): Boolean {
```

Comment:
Is this git history a bit broken? It looks funny.

Reply:
Don't know how it ended up like that